instrumentation/events/lttng-module/net.h

```diff
@@ -8,7 +8,106 @@
 #include <linux/skbuff.h>
```

Comment:
comment about changelog (same as already stated in previous PR): please add a few examples of babeltrace event output (pretty-printed) generated by those events. Do so for the 2 commits.